docs/styles/color.md

```diff
@@ -0,0 +1,41 @@
+# Color
+
+The `color` rule sets the text color of a Widget.
+
+=== "color.py"
+
+    ```python
+    --8<-- "docs/examples/styles/color.py"
+    ```
+
+=== "Output"
+
+    ```{.textual path="docs/examples/styles/color.py"}
+    ```
+
+## CSS
+
+```sass
+/* Blue text */
+color: blue;
+
+/* 20% red text */
+color: red 20%;
+
+/* RGB color */
+color: rgb(100,120,200);
```

Comment:
Might want to mention RGBA, HSL, HSLA? Hex syntax? Maybe that stuff belongs elsewhere? We could say the value should be a "Color", then we link off to all the information for how you can specify a "Color" (otherwise we may end up duplicating explanations across each rule that uses Color values)

```suggestion
/* RGB color */
color: rgb(100,120,200);

/* RGBA color (50% opacity) */
color: rgb(100,120,200, 0.5);  
```